### PR TITLE
Formats and requirements for emote upload

### PR DESCRIPTION
**Pull request by WaterBoiledPizza**
Add the supported formats and requirements PERMANANTLY at the side.
image

### CODE REVIEW

**Review by AnatoleAM — CHANGES_REQUESTED** [review]
Please fix issues with responsiveness

image
image

File: src/views/emote-upload/EmoteUpload.vue [hunk 1]
```diff
@@ -72,23 +89,15 @@
...
-					<span>
-						<div v-if="parentEmote" class="parent-emote">
-							<img :src="getImage(parentEmote.host, ImageFormat.WEBP, 2)?.url" />
-							<div class="as-child-notice">
-								<i18n-t keypath="emote.upload.as_child" tag="p">
-									<span style="font-weight: 600">{{ parentEmote.name }}</span>
-								</i18n-t>
+							<div class="requirements">
+								<h3>Requirements:</h3>
+								<br />- under 7 MB <br />- smaller than 1000 x 1000 pixels <br />- less than 1000 frames
```
Comment (AnatoleAM) [inline src/views/emote-upload/EmoteUpload.vue:94]: Do not use legacy tags such as "`<br />`", use css or paragraph tags instead.

File: src/views/emote-upload/EmoteUpload.vue [hunk 2]
```diff
@@ -161,11 +170,11 @@ if (parentID.value) {
 }
 
 // Formats viewer
-const formatsViewerOpen = ref(false);
+const formatsViewerOpen = ref(true);
 const formatsViewer = ref<HTMLElement | null>(null);
 
 onMounted(() => {
-	onClickOutside(formatsViewer, () => (formatsViewerOpen.value = false));
+	onClickOutside(formatsViewer, () => (formatsViewerOpen.value = true));
```
Comment (AnatoleAM) [inline src/views/emote-upload/EmoteUpload.vue:177]: This is deadcode

**Review by AnatoleAM — CHANGES_REQUESTED** [review]

**Comment by WaterBoiledPizza** [comment]
Change to vertical order when page is less than 1200 px wide, and cleaned up the code accordingly.
chrome_HXD6l67c8d
chrome_pxojlimZrp
chrome_OAKYM4pBkw

File: src/views/emote-upload/EmoteUpload.vue (hunk 1 above)
Comment (WaterBoiledPizza) [reply src/views/emote-upload/EmoteUpload.vue]: can i know why legacy tags should not be used?

Comment (AnatoleAM) [reply src/views/emote-upload/EmoteUpload.vue]: because they're legacy and this is not 2008

File: src/views/emote-upload/EmoteUpload.vue (hunk 2 above)
Comment (WaterBoiledPizza) [reply src/views/emote-upload/EmoteUpload.vue]: done Okayeg

**Review by AnatoleAM — COMMENTED** [review]
I don't think having 2 blocks looks great. Would be best to have the requirements aligned below the upload box

**Pull request closed without merging**